library/ssl_tls13_client.c

```diff
@@ -262,12 +262,6 @@ static int ssl_tls13_generate_and_write_ecdh_key_exchange(
 
     }
 
-    if( own_pubkey_len > (size_t)( end - buf ) )
```

Comment:
Just one thing. In `psa_ssl_status_to_mbedtls` I do not see the translation of PSA_ERROR_BUFFER_TOO_SMALL which would be nice to have here. Would you be ok to add it?